tests/unit/user/payment.test.ts

```diff
@@ -30,22 +35,274 @@ describe ("Payment Tests", ()=>{
         await userController.createUser({
             ...userInfo
         })
+        let userId;
+        try {
+            const user = await User.findOne({
+                email: "michaeltest@gmail.com"
+            }).exec();
+            
+            if (user) {
+                userId = user._id;
+            } else {
+                console.log("No user found with that email");
+            }
+        } catch (err) {
+            console.error(err);
+        }
+     
+        let date = new Date();
+        const paymentInfo = {
+            memberId: userId?.valueOf(),
+            paymentAmount: 20,
+            paymentDate: date,
+            paymentStatus: Status[0].toString()
+        };
+        
+        
+        await paymentController.createPayment({ 
+            ...paymentInfo
+        })
+
+        const paymentInstance = await Payments.findOne({
+            memberId: userId?.valueOf()
+        }).exec()
+        
+        expect(paymentInstance).to.exist;
+        expect(paymentInstance?.paymentDate).to.include(paymentInfo.paymentDate);
+        expect(paymentInstance?.memberId).to.include(paymentInfo.memberId);
+        expect(paymentInstance?.paymentAmount).to.equal(paymentInfo.paymentAmount);
+        expect(paymentInstance?.paymentStatus).to.equal(paymentInfo.paymentStatus);
+        
+        
+})  
+    it ("PaymentController::findUserPaymentHistory finds user and sort them in order by date", async () =>{
+       //creating two different users
+       await Payments.deleteMany({});
```

Comment:
I don't think you need this line twice

Reply:
oh my bad the other one is supposed to be User.deleteMany did not catch